kpops/component_handlers/kafka_connect/connect_wrapper.py

```diff
@@ -57,7 +58,7 @@ async def create_connector(
         if response.status_code == httpx.codes.CREATED:
             log.info(f"Connector {connector_config.name} created.")
             log.debug(response.json())
-            return KafkaConnectResponse(**response.json())
+            return KafkaConnectResponse.model_validate(response.json())
```

Comment:
It's quite confusing to me that `model_validate` has a return value 🫠

Reply:
true, it's not the most intuitive naming. It's Pydantic's builtin constructor classmethod.

Reply:
Sorry, should have been more clear. I didn't want to ask you to change it, just venting about Pydantic naming 😅